08-translations.Rmd

```diff
@@ -0,0 +1,160 @@
+and `stats`) contains a `po` directory that is the central location for cataloguing/translating each package's
+messages.
+
+### .pot files
+
+The `.pot` file is a snapshot of the messages available in a given **domain**. A domain in R typically identifies
+a source package and a source language (either R or C/C++). For example, the file `R-stats.pot`
+(found in the R sources in `./src/library/stats/po`) is a catalogue of all messages produced by R code in the
+`base` package, while `stats.pot` is a catalogue of all messages produced by C code in the `stats` package.
+
+There are two exceptions to the basic pattern described above. The first is the domain for messages produced by
+the C code which is the fundamental backing of R itself (especially, but not exclusively, the C code under
+./src/main). The associated .pot file is R.pot and it is found in `./src/library/base/po`. R-base.pot is a
+normal .pot file because base has a normal `R` directory.
+
+The second is the domain for the Windows R GUI, i.e., the text in the menus and elsewhere in the R GUI program
+available for running R on Windows. These messages are stored in the `RGui.pot` domain, also in the `po`
+directory for `base`, and are most commonly derived from C code found in `./src/gnuwin32`. One reason to keep
+this domain separate is that it is only relevant to one platform (Windows). In particular, Windows has historically
+different character encodings, so that it made more sense for Windows developers to produce translations
+for Windows, since it is non-trivial for non-Windows users to test their translations for the Windows GUI.
+
+#### Generating .pot files
+
+For outside contributors, there's no need to update .pot files -- translators will typically take the R .pot files
+as given and generate .po files. These will be sent along to a language-specific translation maintainer, who then
+compiles them to send to the R Core developer responsible for translations, who finally applies them as a patch.
+
+To emphasize, this section is almost always not needed for contributing translations -- it is here for
+completeness and edification.
+
+### .po files
+
+.po files are the most important artifacts for translators. They provide the (human-readable!) mapping between
+the messages as they appear in the source code and how the messages will appear to users in translated locales.
+
+#### Singular messages
+
+Most messages appear as `msgid`/`msgstr` pairs. The former gives the message as it appears in the code, while the
+latter shows how it should appear in translation. For example, here is an error in German (locale: `de`) informing
+the user that their input must be of class `POSIXt`
+
+```
+msgid "'to' must be a \"POSIXt\" object"
+msgstr "'to' muss ein \"POSIXt\" Objekt sein"
+```
+
+See this in context in the [`R-de.po` source file](https://github.com/r-devel/r-svn/blob/c715d61cb74b3fee2d035faed9b258e86e420b75/src/library/base/po/R-de.po#L537-L538).
```

Comment:
Why are you not linking to the SVN, i.e.,

https://svn.r-project.org/R/trunk/src/library/base/po/R-de.po